coderd/coderdtest/coderdtest_test.go

```diff
@@ -27,3 +30,20 @@ func TestNew(t *testing.T) {
 	_, _ = coderdtest.NewGoogleInstanceIdentity(t, "example", false)
 	_, _ = coderdtest.NewAWSInstanceIdentity(t, "an-instance")
 }
+
+// TestOrganizationMember checks the coderdtest helper can add organization members
+// to multiple orgs.
+func TestOrganizationMember(t *testing.T) {
+	t.Parallel()
+
+	client := coderdtest.New(t, &coderdtest.Options{})
+	owner := coderdtest.CreateFirstUser(t, client)
+
+	second := coderdtest.CreateOrganization(t, client, coderdtest.CreateOrganizationOptions{})
+	third := coderdtest.CreateOrganization(t, client, coderdtest.CreateOrganizationOptions{})
+
+	// Assign the user to 3 orgs in this 1 statement
+	_, user := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID, rbac.ScopedRoleOrgMember(second.ID), rbac.ScopedRoleOrgMember(third.ID))
```

Comment:
This is the result of this change. This line creates a new user and assigns them to 3 organizations. Much easier than doing this manually.